src/components/SendLessonModal/SendLessonModal.stories.tsx

```
@@ -428,3 +428,122 @@ export const WithCustomTitle: Story = () => {
+        schoolId: 'escola-unica',
+      },
+    ],
+    selectedIds: [],
+  },
+  {
+    key: 'turma',
+    label: 'Turma',
+    dependsOn: ['serie'],
+    filteredBy: [{ key: 'serie', internalField: 'schoolYearId' }],
+    itens: [
+      { id: 'turma-unica', name: '9º Ano A - Manhã', schoolYearId: 'serie-unica' },
```

ESLint

[error] Replace `·id:·'turma-unica',·name:·'9º·Ano·A·-·Manhã',·schoolYearId:·'serie-unica'` with `⏎········id:·'turma-unica',⏎········name:·'9º·Ano·A·-·Manhã',⏎········schoolYearId:·'serie-unica',⏎·····`

(prettier/prettier)